pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1537,6 +1537,20 @@ type ObjectStoreSpec struct {
 	// +nullable
 	// +optional
 	Hosting *ObjectStoreHostingSpec `json:"hosting,omitempty"`
+
+	// RGWReadAffinity defines the read affinity policy to optimize the read requests for the RGW clients
+	RGWReadAffinity *RGWReadAffinity `json:"rgwReadAffinity,omitempty"`
```

Comment:
This seems like a `Gateway` setting to me

Reply:
Yes. I've moved it under GatewaySpec